core/point/point.go

```diff
@@ -46,55 +43,46 @@ func New(x, y int64) *Point {
 	}
 }
 
-// X returns the x-value.
+// X method receives a pointer argument of type *Point,
+// and returns the x-value.
 func (pt *Point) X() int64 { return pt.x }
 
-// Y returns the y-value.
+// Y method receives a pointer argument of type *Point,
+// and returns the y-value.
 func (pt *Point) Y() int64 { return pt.y }
 
-// ToSGF converts a pointer-type (immutable) *Point
+// ToSGF method receives a pointer argument of type *Point,
+// and converts the pointer-type (immutable) *Point
 // to an SGF Point (two letter string).
-// The returned value is 0-indexed.
 func (pt *Point) ToSGF() (string, error) {
-	var sgfOut string
-	var err01 = fmt.Errorf("")
-	if (pt.X() <= 51) && (pt.Y() <= 51) {
-		sgfX := string(pointToSgfMap[pt.X()])
-		sgfY := string(pointToSgfMap[pt.Y()])
-		sgfOut = sgfX + sgfY
-		err01 = nil
-	} else {
-		sgfOut = ""
-		err01 = fmt.Errorf("*Point int64 x and y value entries must" +
-			" be greater than or equal to 0, " +
-			"and less than or equal to 51. ")
-		// err01 = errors.New("*Point x and y value entries must be" +
-		// 	" greater than or equal to 0, and less than or equal to 51")
+	if (pt.X() < 0) || (51 < pt.X()) || (pt.Y() < 0) || 51 < (pt.Y()) {
```

Comment:
Won't this always error-out now?

Reply:
Hey, I don't think so because with the previous conjunctive case, ie with &&, it will only error out with both bas conditions met; such as (pt.X() < 0) && (51 < pt.X()), but with the above disjunctive, ie with ||, it will error out with either bad condition.(pt.X() < 0) || (51 < pt.X()), and same with Y().. Does that make sense to you? Let me know if I am missing something.  :  )

Reply:
oh, I see. I would recommend keeping consistent pattern for the conditionals -- I.e., Blah <compare> Number. `if pt.X() < 0 || pt.X() >51 || pt.Y() < 0 || pt.Y() > 56`